xonsh/aliases.py

```diff
@@ -65,9 +67,70 @@ def xexec(args, stdin=None):
         try:
             os.execvpe(args[0], args, denv)
         except FileNotFoundError as e:
-            return "xonsh: " + e.args[1] + ": " + args[0] + "\n"
+            return 'xonsh: ' + e.args[1] + ': ' + args[0] + '\n'
     else:
-        return "xonsh: exec: no args specified\n"
+        return 'xonsh: exec: no args specified\n'
+
+
+def history(args, stdin=None):
```

Comment:
Just to be clear, this command and the other new aliases in this file are something you want to wait on?  Looks like they are just kind of stubbed out for now.